llvm/lib/Target/NVPTX/NVPTXISelDAGToDAG.cpp

```diff
@@ -2182,6 +2182,84 @@ bool NVPTXDAGToDAGISel::tryStoreRetval(SDNode *N) {
   return true;
 }
 
+// Helpers for constructing opcode (ex: NVPTX::StoreParamV4F32_iiri)
+#define getOpcV2H(ty, op0, op1) NVPTX::StoreParamV2##ty##_##op0##op1
```

Comment:
The argument naming structure is a bit confusing. 
In some places like here `opN` are  `i` and `r` tokens to be glued together. 
In others `opN` are apparently values that are used in comparisons.

Perhaps give them distinct names (opKindN/isImmN ?)

Reply:
Good point, fixed.